### PR TITLE
Rearranged job report email body

### PR DESCRIPTION
**Pull request by ralftar**
Remember version bump if not taking in other cahnges at the same time (e.g. https://github.com/vendanor/CloudDump/pull/44)

### CODE REVIEW

**Comment by ralftar** [comment]
My minutes seconds calculation failed. can you have a look at it @jonaski ?

File: scripts/wrapper.sh [hunk 1]
```diff
@@ -458,19 +458,22 @@ fi
...
 
-message="Vendanor CloudDump v${VERSION} Job report
+message="JOB REPORT
 
 Host: ${HOST}
-ID: ${JOBID}
+Result: ${result_text}
+Started: ${time_start_timestamp}
+Completed: $(timestamp)
+Time elapsed: $((time_end - time_start)/60) minutes $((time_end - time_start)%60) seconds
```
Comment (jonaski) [inline scripts/wrapper.sh:467]:
```suggestion
Time elapsed: $(((time_end - time_start)/60)) minutes $(((time_end - time_start)%60)) seconds
```

**Review by jonaski — CHANGES_REQUESTED** [review]

File: scripts/wrapper.sh [hunk 2]
```diff
@@ -458,19 +458,22 @@ fi
...
 Host: ${HOST}
-ID: ${JOBID}
+Result: ${result_text}
+Started: ${time_start_timestamp}
+Completed: $(timestamp)
+Time elapsed: $(((time_end - time_start)/60)) minutes $(((time_end - time_start)%60)) seconds
+
 Script: ${SCRIPTFILENAME}
+ID: ${JOBID}
+Confiugration:
```
Comment (jonaski) [inline scripts/wrapper.sh:471]:
```suggestion
Configuration:
```

**Review by jonaski — CHANGES_REQUESTED** [review]

**Comment by ralftar** [comment]
Can you go ahead and complete this PR and https://github.com/vendanor/CloudDump/pull/44 into a new release @jonaski ?

**Review by jonaski — APPROVED** [review]

**Pull request merged**